package.json

```diff
@@ -10,7 +10,7 @@
     "typings": "typings install",
     "test": "npm run validate",
     "validate": "npm-run-all -p validate-webpack lint test-unit -s check-coverage",
-    "test-unit": "karma start config/karma.conf.js --single-run --browsers PhantomJS",
+    "test-unit": "karma start karma.conf.js --single-run --browsers PhantomJS",
```

Comment:
I think we can leave the reference to `karma.conf.js` out here, as it's picked up by default (just like the `tsconfig.json` in the `tsc`).